ts/Dashboards/EditMode/EditContextMenu.ts

```diff
@@ -142,14 +142,25 @@ class EditContextMenu extends Menu {
         const contextMenu = this;
 
         if (
-            contextMenu.container
+            contextMenu.container &&
+            this.editMode.tools.contextButtonElement
         ) {
             if (visible) {
                 contextMenu.container.style.display = 'block';
                 contextMenu.isVisible = true;
+
+                this.editMode.tools.contextButtonElement.setAttribute(
```

Comment:
`this.editMode.tools.contextButtonElement` is in a few places, I would suggest to assign it into variable and then use it.